client/src/Movies/MovieList.js

```diff
@@ -35,19 +38,32 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
-      <h2>{title}</h2>
+
+      <h2>
+        <Link 
```

Comment:
Great job adding in the Link component! I like the way you used it, as it seems to make the most sense to have it on the movie title. Awesome stuff!